models/model-cards/gnn-fsi-modelcard.md

```diff
@@ -0,0 +1,183 @@
+* Not Applicable
+
+### What is the age balance of the model validation data?
+* Not Applicable
+
+### What is the language balance of the model validation data?
+* Not Applicable
+
+### What is the geographic origin language balance of the model validation data?
+* Not Applicable
+
+### What is the educational background balance of the model validation data?
+* Not Applicable
+
+### What is the accent balance of the model validation data?
+* Not Applicable
+### Describe measures taken to mitigate against unwanted bias.
+* Not Applicable
+## Model Card ++ Explainability Subcard
+
+### Name example applications and use cases for this model. 
+* This model is intended to be used for fraud detection use cases.
+### Fill in the blank for the model technique.
+* This model is intended for developers that want to build and/or customize fraud detection applications using graph neural network.
+
+### Name who is intended to benefit from this model. 
+* This model is intended for users that use transactional data to build fraud detection application.
+
+### Describe the model output.
+* This model outputs fraud probability score b/n (0 & 1). 
+
+### List the steps explaining how this model works. (e.g., )  
+* The model uses a bipartite heterogeneous graph representation as input for `GraphSAGE` for feature learning and `XGBoost` as a classifier. Since the input graph is heterogeneous, a heterogeneous implementation of `GraphSAGE` (HinSAGE) is used for feature embedding.<br>
+
+### Name the adversely impacted groups (protected classes) this has been tested to deliver comparable outcomes regardless of:
+* Not Applicable
+
+### List the technical limitations of the model.
+* This model version requires a transactional data schema with entities (user, merchant, transaction) as requirement for the model.
+
+### What performance metrics were used to affirm the model's performance?
+* Area under ROC curve and Accuracy
+
+### What are the potential known risks to users and stakeholders? 
+* Not Applicable
+
+### What training is recommended for developers working with this model?  If none, please state "none."
+* none
+### Link the relevant end user license agreement 
+* [Apache 2.0](https://github.com/nv-morpheus/Morpheus/blob/branch-23.07/LICENSE)
+
+## Model Card ++ Saftey & Security Subcard
+
+### Link the location of the training dataset's repository (if able to share).
+* [training dataset](models/datasets/training-data/fraud-detection-training-data.csv)
+
+### Is the model used in an application with physical safety impact?
+* No
+
+### Describe physical safety impact (if present).
+* Not Applicable
+
+### Was model and dataset assessed for vulnerability for potential form of attack?
+* No
+### Name applications for the model.
+* Fraud detection in credit card transaction with the defined schema.
+### Name use case restrictions for the model.
+* Only tested for fraud detection in transactional data application
```

Comment:
Same as above (e.g. `The model's use case is restricted to testing the Morpheus pipeline and may not be suitable for other applications.`)